recipe_scrapers/simplywhisked.py

```diff
@@ -0,0 +1,29 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, normalize_string, get_yields
```

Comment:
Are you using these at all? If not, please remove them.

Reply:
I'm sorry about that, I see what I missed with the pre-commit checks now, I removed the unused imports and fixed the formatting and I think everything should be good now!

Reply:
Thanks for the update!